iOS/CFC_Tracker/CFC_Tracker/HTMLDisplayCommunicationHelper.m

```diff
@@ -55,6 +54,7 @@ - (void)postToHost {
     [request setHTTPMethod:@"POST"];
     [request setValue:@"application/json"
    forHTTPHeaderField:@"Content-Type"];
+    request.cachePolicy = NSURLRequestReturnCacheDataElseLoad;
```

Comment:
Is this still required? I thought that you said that the cache doesn't actually work.